monai/utils/enums.py

```diff
@@ -462,15 +462,25 @@ def get_sort_fn(sort_fn):
             )
 
 
-class WSIPatchKeys(StrEnum):
+class PatchKeys(StrEnum):
     """
-    The keys to be used for metadata of patches extracted from whole slide images
+    The keys to be used for metadata of patches extracted from any kind of image
     """
 
     LOCATION = "location"
-    LEVEL = "level"
     SIZE = "size"
     COUNT = "count"
+
+
+class WSIPatchKeys(StrEnum):
```

Comment:
Can we inherit from `PatchKeys` here?

Thanks.

Reply:
Unfortunately, enums cannot be inherited.